### PR TITLE
Headless chrome

### PR DESCRIPTION
**Pull request by mauve**
Asking for feedback.

Initial work to render snapshots using chrome headless rather than phantom.

This right now only works when building from master because the required binaries are not bundled when building.

chrome-headless

### CODE REVIEW

File: pkg/components/renderer/renderer.go [hunk 1]
```diff
@@ -114,7 +91,74 @@ func RenderToPng(params *RenderOpts) (string, error) {
...
+	if !setting.RenderingHeadless {
+		cmdArgs = append(cmdArgs, "headless=false")
+	}
+
+	if setting.RenderingSlowMo > 0 {
+		cmdArgs = append(cmdArgs, fmt.Sprintf("slowmo=%d", setting.RenderingSlowMo))
+	}
+
+	// hehe hardcode node for now
+	return exec.Command("node", cmdArgs...)
```
Comment (bergquist) [inline pkg/components/renderer/renderer.go]: Does this means Grafana will require nodejs to start headless chrome? We cannot have nodejs as an requirement for running Grafana.

Comment (mauve) [reply pkg/components/renderer/renderer.go]: I am a noob, but how do I execute my `screenshot.js` file if not with node?

Comment (torkelo) [reply pkg/components/renderer/renderer.go]: Maybe there is a way to control headless chrome from go? like https://github.com/chromedp/chromedp

**Comment by mauve** [comment]
I will give It a try

**Comment by mauve** [comment]
closed because we are trying to go with chromedp instead

**Pull request closed without merging**